cosmos/eightball.json

```diff
@@ -0,0 +1,46 @@
+{
+  "chainId": "eightball-1",
+  "chainName": "8ball",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/8ball/chain.png",
```

Comment:
Could you change `chainSymbolImageUrl` field?

"https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/8ball/chain.png"
8ball -> eightball